contracts/rosca.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.4.4;
 /*
 *  Possible missing functions
 * - integrity check? to perform check and make sure certain values are available and correct
-* TODO: a way to randomly pick a member that havent got paid this
+*
 */
 contract rosca {
```

Comment:
should we follow the Solidity style guide and name contracts in CapWords?
http://solidity.readthedocs.io/en/develop/style-guide.html#contract-and-library-names